samples/apps/bookinfo/rules/mixer-rule-standard-attributes.yaml

```diff
@@ -47,8 +47,30 @@ spec:
         valueType: TIMESTAMP
       source.uid:
         valueType: STRING
+      source.user:
+        valueType: STRING
       target.uid:
         valueType: STRING
+      destination.uid:
+        valueType: STRING
+      connection.id:
+        valueType: STRING
+      connection.received.bytes:
+        valueType: INT64
+      connection.received.bytes_total:
+        valueType: INT64
+      connection.sent.bytes:
+        valueType: INT64
+      connection.sent.bytes_total:
+        valueType: INT64
+      connection.duration:
+        valueType: DURATION
+      context.protocol:
+        valueType: STRING
+      context.timestamp:
+        valueType: TIMESTAMP
+      context.time:
```

Comment:
Why both time and timestamp? This also looks like a more consequential change than quota update.

Reply:
copied from the API repo.